game/scripts/npc/items/farming/item_greater_arcane_boots.txt

```diff
@@ -91,6 +91,21 @@
         "var_type"                                        "FIELD_INTEGER"
         "creep_bounty_percent"                            "30"
       }
+      "07"
+      {
+      	"var_type"										  "FIELD_INTEGER"
+      	"gold_given_max_percentage"						  "100" 	
```

Comment:
I'm not really sure why this is necessary. Should probably also have a KV for reduction per tick.

Reply:
idk I just figured I should have every value I use as a KV for easy changing at a later date. I will add a reduction per tick KV